docs/integrations/sources/file.md

```diff
@@ -5,71 +5,99 @@ This page contains the setup guide and reference information for the Files sourc
 ## Prerequisites
 
 - URL to access the file
-- Format
-- Reader options
-- Storage Providers
+- File Format
+- Storage Provider
+- Reader Options (Optional)
```

Comment:
I think of pre-requisites as "things you need to go get before starting to setup this connector". In that sense, the only requirement is "A file hosted on SFTP, S3, GCS, HTTPS, or SFTP server"

Reply:
Gotcha, fixed it 👍